rootfs/templates/wrapper

```diff
@@ -150,7 +150,7 @@ function use() {
 		--publish ${GEODESIC_PORT}:${GEODESIC_PORT}
 		--name "${DOCKER_NAME}"
 		--rm
-		--env KUBERNETES_API_PORT=${GEODESIC_PORT})
+		--env GEODESIC_PORT=${GEODESIC_PORT})
```

Comment:
I'm not sure about ports

Reply:
@osterman

Reply:
I support this. I want to generalize it so we can use it for both `kubectl` and `teleport`. The port can be overloaded,

Reply:
Also, I like the consistency where the LHS==RHS env name